canonical-web-and-design/maas-ui
# Add release notification setting

## Done

- Add release notification setting.
- Move general form to typescript. 
- Only send analytics when the value changes.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)

### QA steps

- View /r/settings
- Check that there is an option to toggle release notifications (currently does nothing).

## Fixes

Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/2055.
